internal/hmyapi/apiv1/types.go

```diff
@@ -353,6 +353,8 @@ func newRPCStakingTransaction(tx *types2.StakingTransaction, blockHash common.Ha
 // RPCBlock represents a block that will serialize to the RPC representation of a block
 type RPCBlock struct {
 	Number           *hexutil.Big     `json:"number"`
+	ViewID           *hexutil.Big     `json:"viewID"`
```

Comment:
why do you need these here? It is not used right? only sufficient to add to RPCMarshalBlock?

Reply:
yes, it's not used in RPCMarshalBlock and other places. But seems like we record all the other fields in RPCMarshalBlock here, so I add them here too. Do we want to keep the type RPCBlock?

Reply:
ok. later rpc revamp will take care of this then.